### PR TITLE
Use storagenode_storage_tally rather than accounting_raws 

### PR DESCRIPTION
**Pull request by jenlij**
What: Renames accounting_raws to storagenode_storage_tallies and updates corresponding methods. 

Why: Accounting_raws no longer holds bw data. Renames model to maintain consistency with bandwidth and bucket schemas.

## Code Review Checklist (to be filled out by reviewer)
 - [ ] Does the PR describe what changes are being made?
 - [ ] Does the PR describe why the changes are being made?
 - [ ] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [ ] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [ ] Could the PR be broken into smaller PRs?
 - [ ] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [ ] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [ ] Does any documentation need updating?

### CODE REVIEW

File: pkg/accounting/db.go [hunk 1]
```diff
@@ -50,22 +49,22 @@ type Rollup struct {
...
+	// SaveRollup records at rest tallies and bw rollups to the accounting_rollups table
 	SaveRollup(ctx context.Context, latestTally time.Time, stats RollupStats) error
 	// SaveBucketTallies saves the latest bucket info
 	SaveBucketTallies(ctx context.Context, intervalStart time.Time, bucketTallies map[string]*BucketTally) ([]BucketTally, error)
 	// QueryPaymentInfo queries Overlay, Accounting Rollup on nodeID
 	QueryPaymentInfo(ctx context.Context, start time.Time, end time.Time) ([]*CSVRow, error)
-	// DeleteRawBefore deletes all raw tallies prior to some time
-	DeleteRawBefore(ctx context.Context, latestRollup time.Time) error
+	// DeleteTalliesBefore deletes all storagenode storage tallies prior to some time
+	DeleteTalliesBefore(ctx context.Context, latestRollup time.Time) error
```
Comment (JessicaGreben) [inline pkg/accounting/db.go]: there are bucket tallies and storagenode tallies, if `DeleteTalliesBefore` only deletes the storagenode tallies could we specify that in the name?

File: satellite/satellitedb/migrate.go [hunk 2]
```diff
@@ -601,6 +601,41 @@ func (db *DB) PostgresMigration() *migrate.Migration {
...
+				Action: migrate.SQL{
+					`DROP TABLE storagenode_storage_tallies CASCADE`,
+					`ALTER TABLE accounting_raws RENAME TO storagenode_storage_tallies`,
+					`ALTER TABLE storagenode_storage_tallies DROP COLUMN data_type`,
+				},
+			},
+			{
+				Description: "Retrofit storagenode_bandwidth_rollups into accounting_rollups since April 1, 2019",
+				Version:     18,
+				Action: migrate.SQL{
```
Comment (JessicaGreben) [inline satellite/satellitedb/migrate.go]: can these migrations be one migration? and just add multiple actions statements to migration 17

File: pkg/accounting/tally/tally.go [hunk 3]
```diff
@@ -71,7 +71,7 @@ func (t *Service) Tally(ctx context.Context) error {
 		errAtRest = errs.New("Query for data-at-rest failed : %v", err)
 	} else {
 		if len(nodeData) > 0 {
-			err = t.SaveAtRestRaw(ctx, latestTally, time.Now().UTC(), nodeData)
+			err = t.SaveStoragenodeStorageTallies(ctx, latestTally, time.Now().UTC(), nodeData)
```
Comment (JessicaGreben) [inline pkg/accounting/tally/tally.go]: Any chance we can try to format `SaveStoragenodeStorageTallies` and `SaveBucketTallies`  the same?  

```
if len(nodeData) > 0 {
    err = t.SaveStoragenodeStorageTallies(ctx, latestTally, time.Now().UTC(), nodeData)
    if err != nil {
        errAtRest = errs.New("Saving storage node data-at-rest failed : %v", err)
    }
}
if len(bucketData) > 0 {
    _, err = t.accountingDB.SaveBucketTallies(ctx, latestTally, bucketData)
    if err != nil {
        errBucketInfo = errs.New("Saving bucket storage data failed")
    }
}
```

File: pkg/accounting/db.go [hunk 4]
```diff
@@ -50,22 +49,22 @@ type Rollup struct {
...
-	// SaveAtRestRaw records raw tallies of at-rest-data.
-	SaveAtRestRaw(ctx context.Context, latestTally time.Time, created time.Time, nodeData map[storj.NodeID]float64) error
-	// GetRaw retrieves all raw tallies
-	GetRaw(ctx context.Context) ([]*Raw, error)
-	// GetRawSince retrieves all raw tallies since latestRollup
-	GetRawSince(ctx context.Context, latestRollup time.Time) ([]*Raw, error)
-	// GetStoragenodeBandwidthSince retrieves all storagenode_bandwidth_rollup entires since latestRollup
-	GetStoragenodeBandwidthSince(ctx context.Context, latestRollup time.Time) ([]*StoragenodeBandwidthRollup, error)
-	// SaveRollup records raw tallies of at rest data to the database
+	// SaveSNStorageTallies records the storagenode at rest data and updates LastTimestamp
```
Comment (egonelbre) [inline pkg/accounting/db.go]: Often shortening names shows that structure can show that program structure can be improved. For example if the database interfaces was one of:

```
type StoragenodeAccounting interface { }
type ProjectAccounting interface { }

or. 

type StorageDB interface { }
type BandwidthDB interface { }
```

Comment (jenlij) [reply pkg/accounting/db.go]: that is a good idea. I'll open another ticket for that.

File: satellite/satellitedb/dbx/satellitedb.dbx.postgres.sql [hunk 5]
```diff
@@ -154,10 +145,12 @@ CREATE TABLE storagenode_bandwidth_rollups (
...
-	storagenode_id bytea NOT NULL,
-	interval_start timestamp NOT NULL,
-	total bigint NOT NULL,
-	PRIMARY KEY ( storagenode_id, interval_start )
+	id bigserial NOT NULL,
+	node_id bytea NOT NULL,
+	interval_end_time timestamp with time zone NOT NULL,
+	data_total double precision NOT NULL,
+	created_at timestamp with time zone NOT NULL,
+	PRIMARY KEY ( id )
```
Comment (JessicaGreben) [inline satellite/satellitedb/dbx/satellitedb.dbx.postgres.sql]: do we want `PRIMARY KEY ( storagenode_id, interval_start )` here?

**Comment by jenlij** [comment]
Refactored in https://github.com/storj/storj/pull/1897/

**Pull request closed without merging**